src/app/components/surveillance/complaints/complaints-wrapper.jsx

```diff
@@ -10,7 +11,14 @@ import {
   listing as listingPropType,
 } from 'shared/prop-types';
 
+const useStyles = makeStyles({
+  container: {
+    minHeight: 'calc(100vh - 188px)',
```

Comment:
Why are there (at least) three different px values in this style?

If there's any way to have the same value for all the pages it would make a lot more sense to put it into the util style file and just use that across the board

Reply:
No, each page as a different height due to the content that is provided.